x-pack/plugins/infra/public/index.ts

```diff
@@ -31,3 +31,4 @@ export type InfraAppId = 'logs' | 'metrics';
 // Shared components
 export { LazyLogStreamWrapper as LogStream } from './components/log_stream/lazy_log_stream_wrapper';
 export type { LogStreamProps } from './components/log_stream';
+export type { InfraClientStartExports } from './types';
```

Comment:
@weltenwort Is this how we want to export this?

Reply:
@miltonhultgren if I don't remember bad, I think this export was a suggestion from you

Reply:
Yes, hence why I'm double checking with Felix 🙈 Just want to make sure I'm saying the right thing!

Reply:
LGTM :+1: